Assets/_BForBoss/_Weapons/Scripts/ScriptableObjects/WeaponScriptableObject.cs

```diff
@@ -0,0 +1,25 @@
+using Sirenix.OdinInspector;
+using UnityEngine;
+
+namespace Perigon.Weapons
+{
+    public interface IWeaponProperties
+    {
+        float RateOfFire { get; }
+        Sprite Crosshair { get; }
+        float BulletSpread { get; }
+    }
+    
+    [CreateAssetMenu(fileName = "WeaponProperties", menuName = "PerigonGames/Weapon", order = 1)]
+    public class WeaponScriptableObject : ScriptableObject, IWeaponProperties
+    {
+        [SerializeField] private float _rateOfFire = 0.1f;
+        [SerializeField] private float _bulletSpread = 1f;
+        [PreviewField]
+        [SerializeField] private Sprite _crosshairImage = null;
+
+        public float RateOfFire => _rateOfFire;
+        public float BulletSpread => _bulletSpread;
+        public Sprite Crosshair => _crosshairImage;
```

Comment:
Should we make this nullable? Otherwise it seems like the default crosshair is just not used and is a bit redundant

Reply:
Yeah for sure, i'll update it. 
But I was taking account for. e.g. TBD - when you wall run, your crosshair changes to default, and then when u jump off it goes back.  
Just something that `might` happen